corehq/messaging/management/commands/export_conditional_alerts.py

```diff
@@ -0,0 +1,262 @@
+    schedule_type = SIMPLE_SMS_DAILY_SCHEDULE_WITH_TIME
+    time = jsonobject.TimeProperty()
+    message = jsonobject.DictProperty(six.text_type)
+    total_iterations = jsonobject.IntegerProperty()
+    start_offset = jsonobject.IntegerProperty()
+    start_day_of_week = jsonobject.IntegerProperty()
+    extra_options = jsonobject.ObjectProperty(ExtraSchedulingOptions)
+    repeat_every = jsonobject.IntegerProperty()
+
+
+class SimpleSMSAlertSchedule(jsonobject.JsonObject):
+    schedule_type = SIMPLE_SMS_ALERT_SCHEDULE
+    message = jsonobject.DictProperty(six.text_type)
+    extra_options = jsonobject.ObjectProperty(ExtraSchedulingOptions)
+
+
+def open_for_json_write(path):
+    # json.dumps returns bytes in python2, but unicode in python3
+    if six.PY2:
+        return open(path, 'wb')
+
+    return io_open(path, 'w', encoding='utf-8')
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument('domain')
+
+    def get_json_rule(self, rule):
+        json_rule = SimpleSchedulingRule(
+            name=rule.name,
+            case_type=rule.case_type,
+        )
+
+        for criterion in rule.memoized_criteria:
+            definition = criterion.definition
+            if not isinstance(definition, MatchPropertyDefinition):
+                raise CommandError(
+                    "Rule %s references currently unsupported criterion definition for export. "
+                    "Either add support to this script for unsupported criteria or exclude rule "
+                    "from export." % rule.pk
+                )
+
+            json_rule.criteria.append(MatchPropertyCriterion(
+                property_name=definition.property_name,
+                property_value=definition.property_value,
+                match_type=definition.match_type,
+            ))
+
+        if len(rule.memoized_actions) != 1:
+            raise CommandError(
+                "Expected exactly one action for rule %s. This is an unexpected configuration. "
+                "Was this rule created with the UI?" % rule.pk
+            )
+
+        action = rule.memoized_actions[0].definition
+        if not isinstance(action, CreateScheduleInstanceActionDefinition):
+            raise CommandError(
+                "Expected CreateScheduleInstanceActionDefinition for rule %s. This is an unexpected "
+                "configuration. Was this rule created with the UI?" % rule.pk
+            )
+
+        for recipient_type, recipient_id in action.recipients:
+            if recipient_type not in (
+                CaseScheduleInstanceMixin.RECIPIENT_TYPE_SELF,
+                CaseScheduleInstanceMixin.RECIPIENT_TYPE_CASE_OWNER,
+                CaseScheduleInstanceMixin.RECIPIENT_TYPE_LAST_SUBMITTING_USER,
+                CaseScheduleInstanceMixin.RECIPIENT_TYPE_PARENT_CASE,
+                CaseScheduleInstanceMixin.RECIPIENT_TYPE_CUSTOM,
+            ):
+                raise CommandError(
```

Comment:
It might be helpful if this lists the recipient types it's expecting

Reply:
i'd rather not complicate the error messages too much - this is most likely a one-time script that's just going to be used by me, but in the event that any other dev uses this later they could just look at the script to see what's supported